platform/rct/Project/STM32_FLASH.ld

```diff
@@ -36,8 +36,8 @@ ENTRY(Reset_Handler)
 _estack = 0x20000000 + 40K;    /* end of 40K RAM */
 
 /* Generate a link error if heap and stack don't fit into RAM */
-_Min_Heap_Size = 0x200;      /* required amount of heap  */
-_Min_Stack_Size = 0x400; 	/* required amount of stack */
+_Min_Heap_Size = 0x100;      /* required amount of heap  */
+_Min_Stack_Size = 0x100; 	/* required amount of stack */
```

Comment:
This is the stack and "heap" (such as it is) used by main() before freeRTOS fires up.  Once the scheduler is started, we don't need it anymore.  See: http://www.freertos.org/FAQMem.html#RAMReduce

I put "heap" in quotes since it's not really used as heap with a heap manager.  In fact, these two variables are only used further down in the linker script to ensure there's enough space left over, and that's calculated as _Min_Heap_Size + _Min_Stack_Size. 

Net-net, main() gets 0x100 + 0x100 (512 bytes) for a stack upon startup, which is never used again after the scheduler starts.